crates/wysiwyg/src/dom/dom_list_methods.rs

```diff
@@ -148,4 +261,11 @@ mod test {
         model.state.dom.remove_list(&first_handle);
         assert_eq!(tx(&model), text);
     }
+
+    // TODO: move this to a more globally usable location if needed
+    fn ds(dom: &Dom<Utf16String>) -> String {
```

Comment:
There is already a `restore_whitespace` fn in `testutils_composer_model.rs` that does a similar conversion (sorry, it's not the most intuitive name). We could replace it with this or viceversa?

Reply:
Actually the equivalent of this is just the last step of `to_example_format` on the composer model (a.k.a `tx`). I set this temporarily but I think `dom` should have it's own `to_example_format` and it's own shortcut for the tests, as it's not really correct to use composer models on the DOM layer tests.